ncr/src/test/kotlin/com/hartwig/actin/personalization/ncr/datamodel/TestNcrRecordFactory.kt

```diff
@@ -17,7 +17,9 @@ object TestNcrRecordFactory {
     fun minimalDiagnosisRecord(): NcrRecord {
         val baseRecord = minimalRecord()
         return baseRecord
-            .copy(identification = baseRecord.identification.copy(epis = DIAGNOSIS_EPIS))
+            .copy(identification = baseRecord.identification.copy(epis = DIAGNOSIS_EPIS),
```

Comment:
`identification = ...` should be moved to the next line:
```suggestion
            .copy(
                identification = baseRecord.identification.copy(epis = DIAGNOSIS_EPIS),
```